COLLABORATOR_GUIDE.md

```diff
@@ -85,7 +93,6 @@ where a pull request:
 
 The Website Admin group should serve as the final arbiter where required.
```

Comment:
Remove "Admin" here as well

Reply:
Thanks, a good catch! I've fixed that.

Reply:
Thanks, I've removed all the "Admin" :) @fhemberger